src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,50 @@
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+
+type Message = {
+    text: string;
+    type: string;
+    action: string;
+};
+
+type Button = {
+    text?: string;
+    tooltip?: string;
+    disabled?: boolean;
+    hidden?: boolean;
+    data?: unknown[];
```

Comment:
Would be nice to know how is the data structure of this array. Is it possible to check during runtime?